package.json

```diff
@@ -41,13 +41,14 @@
     "grunt-contrib-jshint": "^1.0.0",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-env": "latest",
+    "grunt-if": "https://github.com/binarymist/grunt-if/tarball/master",
     "grunt-jsbeautifier": "^0.2.12",
     "grunt-mocha-test": "^0.12.7",
     "grunt-nodemon": "^0.4.2",
-    "grunt-if": "https://github.com/binarymist/grunt-if/tarball/master",
     "grunt-npm-install": "^0.3.0",
     "grunt-retire": "^0.3.12",
     "mocha": "^2.4.5",
+    "opn": "^4.0.2",
```

Comment:
Since `opn` is expected to ever be executed in a development environment, I think we should keep it as a dev dependency, even if code has a direct dependency on this module.

Typically, users running this locally would have to `npm install` before they `npm start`, so dev dependencies will be installed as well.